README.md

```diff
@@ -14,39 +15,76 @@ A Discord and Fosscord client implemented directly without Discord API.
     webcord.url = "github:fufexan/webcord-flake";
   };
 
-  outputs = { self, nixpkgs, webcord, ... }@inputs: {
+  outputs = {self, nixpkgs, webcord, ...} @ inputs: {
     # for NixOS
     nixosConfigurations.HOSTNAME = nixpkgs.lib.nixosSystem {
-      specialArgs = inputs;
+      specialArgs = {inherit inputs;};
       # ...
     };
 
-    # for Home-Manager
-    homeConfigurations.USER@HOST = home-manager.lib.homeManagerConfiguration {
-      extraSpecialArgs = inputs;
+    # for Home Manager
+    homeConfigurations."USER@HOSTNAME" = home-manager.lib.homeManagerConfiguration {
+      extraSpecialArgs = {inherit inputs;};
       # ...
     };
   };
+}
 ```
 
-```nix
-# configuration.nix or home.nix
-{ pkgs, webcord, ... }:
+> Don't forget to replace `HOSTNAME` with your hostname, or `USER@HOSTNAME` with your profile name!
 
-{
+#### In your `configuration.nix` or `home.nix`:
+
+```nix
+{pkgs, inputs, ...}: {
   environment.systemPackages = [ # or home.packages
-    webcord.packages.${pkgs.system}.default
+    inputs.packages.${pkgs.system}.default
     # ...
   ];
 }
 ```
 
-Don't forget to replace `HOSTNAME` with your hostname, or `USER@HOST` with your profile name!
+### Merging with `pkgs`
+
+Optionally, if you import `nixpkgs` manually (and don't use `legacyPackages`), you can use `pkgs.webcord` as if it were any other package.
```

Comment:
```suggestion
Optionally, if you import `nixpkgs` manually (and don't use `legacyPackages`), you can use `pkgs.webcord` as if it were any other package via an overlay.
```

Reply:
Why is it "via an overlay"? Do overlay packages behave any different? They're still just `callPackage` in the end, afaik

Reply:
Yes, but since it does not come from `nixpkgs` it has to be explicitly added. Just stating the obvious (to us) in order not to confuse the users.

Reply:
Not ready to merge yet. I want to revise this sentence because I think you're right, it needs clarification.